.github/README.md

```diff
@@ -2,7 +2,7 @@ MineStat
 ========
 
 [![AppVeyor build status](https://img.shields.io/appveyor/ci/ldilley/minestat?label=AppVeyor%20build%20status)](https://ci.appveyor.com/project/ldilley/minestat)
-[![Travis CI build status](https://img.shields.io/travis/com/FragLand/minestat?label=Travis%20CI%20build%20status)](https://travis-ci.com/FragLand/minestat)
+[![CodeQL build status](https://github.com/FragLand/minestat/actions/workflows/codeql.yml/badge.svg?branch=master)](https://github.com/FragLand/minestat/actions/workflows/codeql.yml)
```

Comment:
I realize that this PR has not been merged yet (so the SVG path does not currently exist in master). However, viewing this file after merging seems like it may result in a broken image. The URL, `https://github.com/FragLand/minestat/actions/workflows/codeql.yml/badge.svg`, references `badge.svg`, which is not present in this PR. My concern is that the `codeql.yml/badge.svg` path would not work considering that `codeql.yml` is a file and not a directory. Or is this format correct and `badge.svg` gets treated as an argument so that GitHub markdown and/or Actions and/or CodeQL can render it properly?

Reply:
Hi, [according to the documentation](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge), the URL format for the badge is:
`https://github.com/<OWNER>/<REPOSITORY>/actions/workflows/<WORKFLOW_FILE>/badge.svg`
so the change should be correct :)